tools/fraken/go.sum

```diff
@@ -0,0 +1,2 @@
+github.com/hillu/go-yara/v4 v4.2.3 h1:Gazusex0rkL8NCMjc2vXqKBIHZ4kiBlLdMt2DuvhL4A=
```

Comment:
I'm assuming this is the hash of the dependency package, but not sure.   Can you think of any good ways we can keep track of this so that the version gets updated somewhat regularly?  Since it's buried in this tools dir, I'm guessing we won't think to update it very often.  We could potentially add a step in the release process to check or something I suppose.

Reply:
This is an automatically generated file. We can run `go get` to keep it up to date every once in a while but I'm not sure how necessary it will be unless new features are added....adding a step to the release process would modify the file and require another commit & push if changes are made. Not sure if that's what we want.